index.html

```diff
@@ -340,5 +340,27 @@ <h1>
         </div>
       </div>
     </section>
+    <div class="form-div">
+      <form action="" class="form">
+        <h2>Contact me</h2>
```

Comment:
- [ ] Kindly make sure your form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) as stated in the project requirement 🙏 .

Reply:
can you please check my last commit, i have made the inputs require.